va/va.h

```diff
@@ -882,6 +882,13 @@ typedef enum
      *  this setting also could be update by \c VAContextParameterUpdateBuffer
      */
     VAConfigAttribContextPriority       = 41,
+    /** \brief AV1 decoding features.  Read-only.
+     *
+     * This attribute describes the supported features of an
+     * AV1 decoder configuration.  The value returned uses the
+     * VAConfigAttribValDecAV1Features type.
+     */
+    VAConfigAttribValDecAV1Features    = 42,
```

Comment:
The attribute type and returned value type should have different names.
"Val" in the name here should be removed.